packages/server/lib/cloud/protocol.ts

```diff
@@ -58,39 +55,46 @@ class ProtocolManagerImpl implements ProtocolManager {
     this.protocol = await setupProtocol(url)
   }
 
-  async connectToBrowser (options) {
+  async connectToBrowser (cdpClient) {
     debug('connecting to browser for new spec')
 
-    return this.protocol?.connectToBrowser(options)
+    return this.protocol?.connectToBrowser(cdpClient)
   }
 
-  addRunnables (runnables) {
-    this.protocol?.addRunnables(runnables)
+  async addRunnables (runnables) {
```

Comment:
and this function needs to be async even though it doesn't `await` anything? DId I understand that correctly or can we remove the `async` function modifier?